tests/core/usage_statistics/test_anonymizer.py

```diff
@@ -42,3 +42,108 @@ def test_anonymizer_consistent_salt():
     assert anon_name_1 == anon_name_2
     assert len(anon_name_1) == 32
     assert len(anon_name_2) == 32
+
+
+class BaseTestClass:
+    pass
+
+
+class TestClass(BaseTestClass):
+    pass
+
+
+class MyCustomTestClass(TestClass):
+    pass
+
+
+class SomeOtherClass:
+    pass
+
```

Comment:
Non-blocking request: could we add a comment saying that these empty classes are used for testing-only?